views/site/partials/index/_jumbo.php

```diff
@@ -27,7 +27,7 @@
         <div class="row">
             <div class="col-md-12">
                 <div class="dashed-heading-jumbo">
-                    <span><a href="<?= Url::to(['doc/guide/2.0/en/start-installation']) ?>" class="btn">Get Started</a> </span>
+                    <span><a href="<?= Url::to(['doc/guide/2.0/en/start-installation']) ?>" class="btn">Getting Started</a> </span>
```

Comment:
This one invites you to start hacking. I don't think it's -ing, right?

Reply:
`Get Started` is an action `Getting Started` is a statement - I prefer the call to action.

Reply:
Okay, will revert. Getting started is more traditional, but not a big deal, both ways of saying it are correct IMHO.